Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICS/t_solid_density.md

```diff
@@ -0,0 +1 @@
+Reference solid density. It changes with temperature difference.
```

Comment:
reference density is constant. The density then changes from this reference value.